openedx/features/course_experience/templates/course_experience/course-message-fragment.html

```diff
@@ -0,0 +1,75 @@
+## mako
+
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+
+<%!
+from django.conf import settings
+from django.utils.http import urlquote_plus
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.js_utils import js_escaped_string
+from openedx.core.djangolib.markup import HTML
+from django.core.urlresolvers import reverse
+%>
+
+<%
+# set doc language direction
+from django.utils.translation import get_language_bidi
+is_rtl = get_language_bidi()
+%>
+
+<%block name="content">
+% if user_access['is_anonymous']:
+    <div class="course-message grid-manual">
```

Comment:
I'm thinking that there should be a helper function for generating these messages. We only have three now and there is already a lot of boilerplate, and it will get worse when we implement all the future notifications.

In fact, I wonder if we could reuse my user_messages API. Then the same code that is generating the banner message could also generate the in-context message.